app/src/hvr/java/com/igalia/wolvic/ContentHolderFragment.java

```diff
@@ -4,7 +4,7 @@
 import android.app.FragmentTransaction;
 import android.content.SharedPreferences;
 import android.os.Bundle;
-import androidx.preference.PreferenceManager;
+import android.preference.PreferenceManager;
```

Comment:
I'm not sure why we are changing androidx.preference.PreferenceManager back to the deprecated android.preference.PreferenceManager.

https://developer.android.com/reference/kotlin/android/preference/PreferenceManager: android.preference.PreferenceManage was deprecated in API level 29.
Use the [AndroidX](https://developer.android.com/jetpack/androidx.html) [Preference Library](https://developer.android.com/reference/androidx/preference/package-summary.html) for consistent behavior across all devices. For more information on using the AndroidX Preference Library see [Settings](https://developer.android.com/guide/topics/ui/settings.html).